src/Nethermind/Nethermind.Network/NetworkStorage.cs

```diff
@@ -17,10 +20,13 @@ namespace Nethermind.Network
 {
     public class NetworkStorage : INetworkStorage
     {
+        public static LruKeyCache<IPAddress> NodesFilter { get; } = new(2048, "Ip Filter");
```

Comment:
Please refrain from static. It breaks encapsulation.

Reply:
Readonly instance